### PR TITLE
Fix point cloud jitter and quantization issues for additive refinement pnts

### PR DESCRIPTION
**Pull request by DStradley**
Fixes https://github.com/iTwin/itwinjs-core/issues/5333

Investigating problems with a user point cloud showed jitter and gaps in the displayed data, both indicating quantization issues.  The point cloud was PNTS format, but using additive refinement.  Also, the first 13 pnts files were unquantized, but all the rest were quantized.

Found that we were quantizing the unquantized tiles, so first changed that to generate and use unquantized point cloud graphics (for PNTS), but that by itself did not solve the problem.  Investigating further, I found that the RTC_CENTER in these tiles was large enough to cause quantization issues the way it was added into the vertex shader points.  I removed that offset from the vertex shader points, then added it into a transform for the tile instead, and that resolved the problem.

Had to adjust batch range by RTC_CENTER since it is in the transform now (for locate, etc.).  Also had to create a range for unquantized points for the voxelSize calculation, since the qParam range (scale of 1) would not work for that.

TODO
- [x] Run ImageTests

### CODE REVIEW

**Comment by DStradley** [comment]
ImageTests ran and the only differences were considered inconsequential ones from now supporting unquantized pnts data.

**Review by pmconne — COMMENTED** [review]
Run `rush change` and add the files created in common/changes.

File: common/api/core-frontend.api.md [hunk 1]
```diff
@@ -8706,7 +8706,10 @@ export class ReadonlyTileUserSet extends ReadonlySortedArray<TileUser> {
 }
 
 // @internal
-export function readPointCloudTileContent(stream: ByteStream, iModel: IModelConnection, modelId: Id64String, _is3d: boolean, range: ElementAlignedBox3d, system: RenderSystem): Promise<RenderGraphic | undefined>;
+export function readPointCloudTileContent(stream: ByteStream, iModel: IModelConnection, modelId: Id64String, _is3d: boolean, range: ElementAlignedBox3d, system: RenderSystem): Promise<{
+    graphic: RenderGraphic | undefined;
```
Comment (pmconne) [inline common/api/core-frontend.api.md:8710]: Did you run `lint`? I suspect it's going to want commas in place of semicolons.

**Review by pmconne — APPROVED** [review]

File: common/api/core-frontend.api.md (hunk 1 above)
Comment (DStradley) [reply common/api/core-frontend.api.md:8710]: Ran lint and fixed the errors it found.  It said nothing about this though, and I am still getting the extract-api error.

**Comment by pmconne** [comment]
> I am still getting the extract-api error

The error is telling you what you need to do to fix it.

**Comment by DStradley** [comment]
I did "npm run extract-api" in the core\extension folder (step 1), and it generated no modified files anywhere, so can't do step 2.

**Comment by DStradley** [comment]
Don't know what to do about the repeatable MacOS security error.

**Comment by pmconne** [comment]
> Don't know what to do about the repeatable MacOS security error.

As yet there is [no fix available](https://github.com/Leonidas-from-XIV/node-xml2js/issues/663). Nothing to do with your changes.

**Comment by DStradley** [comment]
@Mergifyio backport release/3.7.x

**Pull request merged**